tests/regression/test_embedded_sphere.py

```diff
@@ -17,7 +16,6 @@ def integrate_sphere_area(radius=1, refinement=2):
     return np.abs(assemble(f * dx) - exact) / exact
 
 
-@longtest
```

Comment:
Has this test ever been "long"? Commit message says that the 8 tests run in 11 seconds.